pkgs/applications/science/logic/verit/default.nix

```diff
@@ -1,24 +1,21 @@
-{ stdenv, fetchurl, gmp, flex, bison }:
+{ stdenv, fetchurl, autoreconfHook, gmp, flex, bison }:
 
 stdenv.mkDerivation rec {
   name = "veriT-${version}";
-  version = "201410";
+  version = "201506";
 
   src = fetchurl {
     url = "http://www.verit-solver.org/distrib/${name}.tar.gz";
-    sha256 = "0b31rl3wjn3b09jpka93lx83d26m8a5pixa216vq8pmjach8q5a3";
+    sha256 = "1cc9gcspw3namkdfypkians2j5dn224dsw6xx95qicad6033bsgk";
   };
 
-  buildInputs = [ gmp flex bison ];
+  nativeBuildInputs = [ autoreconfHook flex bison ];
+  buildInputs = [ gmp ];
 
-  enableParallelBuilding = false;
```

Comment:
Can you keep this line please? If it was there it was to point parallel building is not suitable for this package. Removing it makes people think it could be enabled.